CHANGELOG.md

```diff
@@ -6,9 +6,20 @@ This library provides official bindings to the Urban Airship SDK, as well as sam
 
 ### Release Notes
 
+Versions 10.0.0 (.NETStandard & PCL), 10.0.1 (Android), 11.0.0 (iOS)
+====================================================================
+- Update Android SDK to 10.0.1
```

Comment:
before this is released could you bump this to 10.0.2

Reply:
Will do.

Reply:
Done.